social/apps/django_app/default/fields.py

```diff
@@ -11,7 +11,7 @@
     from django.utils.encoding import smart_text
 
 
-class JSONField(six.with_metaclass(models.SubfieldBase, models.TextField)):
+class JSONField(models.TextField):
```

Comment:
@johnbaldwin Because django.db.models.SubfieldBase has been deprecated in Django 1.9, now we must use Django JSONField, I've cherry picked this commit from the original PR:
https://github.com/omab/python-social-auth/pull/813